server/controllers/dashboardController.js

```diff
@@ -0,0 +1,51 @@
+import Joi from '@hapi/joi';
+import Board from '../models/boardModel';
+import Dashboard from '../models/dashboardModel';
+import { buildResponse } from '../utils/helpers';
+import constants from '../config/constants';
+
+function validateBoard(request) {
```

Comment:
breaking the principle of least access, if you require only name, pass only name

Reply:
a request is an object which only contains name and lifecycle, in future if any field added then we have to just change the JOI validation object.